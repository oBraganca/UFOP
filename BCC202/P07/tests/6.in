121 60 49 106 91 102 29 109 40 18 66 119 58 46 11 12 3 75 28 85 5 110 20 61 44 53 48 86 90 79 104 115 27 23 8 76 39 45 19 55 87 51 84 31 95 89 4 30 56 57 38 82 59 68 94 36 33 111 24 10 114 78 16 42 63 96 43 83 121 93 13 32 105 54 80 101 64 35 120 7 47 117 15 6 67 69 52 1 71 72 74 70 118 50 81 100 108 14 112 73 88 41 77 2 25 62 99 9 34 98 107 65 116 22 21 37 92 113 103 26 97 17
77 72 19 60 39 24 59 12 50 71 51 55 4 32 26 8 43 2 75 31 52 6 33 1 76 35 61 74 34 57 3 30 17 15 64 56 70 41 48 68 25 7 42 62 47 73 44 23 5 67 49 21 58 10 77 69 13 22 9 38 65 36 63 46 18 40 20 11 29 27 54 53 37 16 14 66 28 45
20 17 15 18 7 4 12 10 8 19 3 9 6 13 16 20 1 2 14 11 5
91 63 22 32 74 40 21 68 51 24 23 4 36 7 82 27 48 1 61 73 89 55 5 29 87 69 37 71 38 85 80 18 65 57 16 41 13 75 91 86 44 49 14 79 60 20 9 26 72 46 90 78 47 84 43 50 19 67 39 15 2 6 66 12 3 45 64 42 31 10 11 56 81 52 34 53 83 54 8 17 33 30 58 88 28 62 59 77 35 25 76 70
16 15 10 9 2 6 5 3 8 7 1 4 12 16 11 13 14
13 1 13 7 2 11 5 10 9 4 6 12 8 3
175 107 72 146 47 174 74 28 42 160 16 102 159 148 29 55 64 110 85 145 150 127 99 46 84 71 158 97 93 116 167 41 12 152 173 11 136 10 39 123 26 7 25 114 70 69 13 115 61 157 40 169 52 57 108 138 88 120 105 155 106 6 20 82 35 98 112 51 144 149 48 86 161 27 23 60 63 65 122 134 165 101 58 54 5 17 59 164 49 77 133 154 139 62 126 44 113 24 140 80 4 175 109 83 36 166 53 43 153 91 124 117 38 162 33 68 19 8 171 129 142 168 100 125 81 22 37 170 156 172 104 67 121 89 66 119 87 118 147 79 143 15 34 132 45 73 92 111 1 128 2 151 163 56 95 137 131 75 130 9 50 103 18 14 32 21 135 94 141 96 90 3 78 76 30 31
161 82 13 2 22 115 20 63 130 69 137 21 104 1 66 77 33 75 12 157 124 40 89 161 38 128 155 94 7 46 102 100 18 93 98 45 43 57 80 111 30 70 117 39 60 81 141 29 26 103 133 54 50 160 126 110 158 65 114 109 34 119 47 48 67 53 132 24 123 68 55 61 96 58 19 121 10 91 56 101 156 74 86 14 3 9 37 129 143 23 6 36 76 95 71 90 148 83 59 127 120 138 150 27 147 144 87 42 140 154 112 97 113 136 153 108 146 134 105 118 28 88 152 25 85 116 92 8 122 84 4 131 78 125 159 149 106 142 107 15 5 72 145 16 17 51 99 79 35 31 73 49 11 135 64 44 32 139 151 52 41 62
20 1 3 7 10 15 2 11 13 6 20 16 12 19 5 9 17 14 4 8 18
176 7 160 118 16 39 112 101 137 166 169 25 66 81 3 53 41 20 65 42 22 103 97 37 172 151 135 98 161 12 111 47 63 43 147 11 131 129 163 58 107 54 106 36 132 162 55 149 13 30 154 168 130 46 26 165 52 27 152 157 104 59 143 176 32 56 49 105 164 2 94 14 171 122 75 90 91 95 70 5 109 170 45 35 88 127 57 19 92 148 60 48 85 6 61 153 139 72 110 146 23 21 93 64 82 87 1 155 144 125 69 68 140 44 115 133 119 77 50 124 108 62 29 10 113 78 167 142 31 102 116 145 150 134 71 83 67 76 89 33 175 40 84 51 28 136 120 158 123 38 73 17 117 173 24 74 9 121 4 114 34 141 156 15 159 18 174 128 138 100 99 96 86 79 8 126 80
104 61 103 55 6 96 8 16 95 33 38 32 74 84 21 59 19 49 89 30 14 77 42 36 25 31 81 35 40 75 93 9 79 56 67 41 91 29 39 46 22 73 15 83 54 48 98 47 51 69 2 11 82 58 1 63 78 17 68 97 4 23 24 65 45 44 92 85 18 72 57 27 3 20 80 88 104 64 43 101 86 71 13 10 7 62 28 94 34 76 102 70 50 5 66 52 90 87 60 12 99 53 100 26 37
169 165 26 16 164 37 135 158 34 167 94 73 130 162 7 24 131 115 81 25 31 168 163 105 129 69 54 142 139 2 156 78 35 33 3 70 60 36 83 68 154 38 107 22 116 160 109 104 58 53 71 85 5 30 144 15 101 80 19 161 18 28 42 110 57 4 90 118 45 112 157 98 150 72 95 56 108 87 11 99 102 96 29 93 55 64 14 59 82 61 148 159 146 63 20 86 52 91 114 79 21 138 132 106 128 126 149 117 23 84 145 122 140 151 77 103 44 136 6 67 13 32 50 147 74 66 133 40 39 88 152 92 43 137 51 169 9 27 119 153 100 47 10 41 166 1 121 125 65 12 111 113 89 123 155 62 76 97 124 75 46 8 143 120 127 48 134 141 49 17
140 18 17 121 75 60 126 138 94 63 90 66 55 97 83 114 92 30 81 47 64 110 79 67 24 109 118 82 86 49 37 93 38 69 19 13 22 50 15 72 59 41 113 137 122 58 117 2 33 91 29 3 42 116 130 84 68 99 16 136 40 6 61 7 78 57 106 77 128 98 56 28 74 70 112 31 89 105 20 62 54 27 123 131 32 46 102 100 104 133 134 107 125 23 44 9 108 135 140 80 26 4 119 14 25 35 8 11 111 120 115 21 73 5 51 48 45 52 1 43 10 88 76 129 132 87 103 124 139 34 65 127 71 36 53 95 39 96 101 85 12
96 87 38 53 86 80 48 71 26 65 29 76 22 85 95 3 32 13 30 89 82 25 79 64 33 47 52 18 42 27 91 56 49 15 2 44 77 36 69 66 57 54 96 11 17 45 50 35 61 43 70 41 59 40 24 81 67 84 74 90 94 83 28 14 88 72 34 60 58 9 63 5 62 10 6 1 92 31 78 8 37 68 21 19 51 73 7 75 46 20 55 12 16 23 39 93 4
67 9 31 55 35 67 64 37 25 22 29 8 61 36 24 65 12 48 57 45 17 18 23 41 26 1 52 3 4 56 2 10 40 58 43 19 51 53 46 6 66 38 7 39 34 28 27 54 33 44 32 21 15 50 60 5 11 49 16 13 62 30 42 14 47 20 59 63
105 71 40 24 37 75 87 52 45 70 54 68 9 94 8 84 99 31 56 32 41 34 79 93 27 78 85 90 26 89 64 58 61 19 55 103 20 104 22 105 48 97 28 100 66 77 4 59 67 51 65 39 12 81 44 2 15 43 5 17 23 42 47 69 91 3 101 76 33 88 10 53 13 73 82 57 7 92 98 14 86 80 16 1 50 35 95 29 63 18 60 74 30 25 83 49 21 96 62 11 72 46 102 6 38 36
94 77 14 69 13 12 94 57 24 18 86 21 5 46 11 39 25 51 37 22 2 49 42 72 54 78 35 73 56 43 89 45 93 68 81 58 71 64 80 19 63 38 70 52 61 76 53 23 84 91 33 34 74 4 6 59 27 47 83 1 55 87 16 29 75 10 40 79 85 26 90 36 8 67 48 32 44 60 28 15 50 30 9 7 20 66 3 41 92 88 82 65 62 31 17
21 16 14 5 11 19 10 15 18 2 20 3 8 7 6 4 21 1 12 17 13 9
74 27 68 28 13 72 20 44 16 47 1 39 42 26 55 49 64 63 32 53 66 4 48 30 74 31 69 17 11 24 10 40 9 22 50 21 19 25 54 38 14 62 59 23 15 3 46 51 73 34 56 2 12 37 67 61 70 7 29 18 57 35 45 41 5 6 33 43 71 52 36 65 8 58 60
111 48 31 77 90 76 9 105 89 68 92 83 102 73 74 12 63 71 4 18 101 49 100 15 20 64 37 110 99 51 61 94 45 66 65 24 22 2 11 28 78 27 84 93 5 106 98 111 38 3 26 16 56 54 109 8 108 17 35 82 33 59 46 44 107 14 47 36 34 21 55 53 79 41 50 30 91 80 19 60 72 104 69 96 103 43 87 7 1 95 25 29 39 75 81 85 10 52 67 97 42 62 86 70 88 40 32 13 58 57 23 6
0
