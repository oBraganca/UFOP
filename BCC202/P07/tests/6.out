Carlos
Marcelo
Marcelo
Carlos
Marcelo
Marcelo
Marcelo
Carlos
Marcelo
Carlos
Marcelo
Carlos
Marcelo
Carlos
Carlos
Marcelo
Marcelo
Marcelo
Carlos
Carlos
